pkg/labels/labels.go

```diff
@@ -28,6 +28,8 @@ type Labels interface {
 
 	// Get returns the value for the provided label.
 	Get(label string) (value string)
+
+	String() string
```

Comment:
this is a convenience interface update, so that any `Labels` support hashing.   should generally work since there is a String rep, but havent tested it under all conditions